airbyte-integrations/connectors/source-file/file_source/spec.json

```diff
@@ -0,0 +1,24 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/template-python-source",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Csv source Spec",
+    "type": "object",
+    "required": ["data_url"],
+    "additionalProperties": false,
+    "properties": {
+      "data_url": {
+        "type": "string",
+        "description": "The path to the file to be replicated."
+      },
+      "reader": {
+        "type": "string",
+        "description": "The reader from pandas library to use"
```

Comment:
can we obscure pandas? the idea is we'd like a fairly non-technical person to be able to fill out this configuration. so asking them if they have csv versus json or something like that is okay, but i think exposing pandas might be going too technical. it also locks us into a specific implementation forever.

Reply:
looks like this should be pretty easy to do though. you can just have an enum that maps from human readable names to pandas names. as long as we provide an enum with human readable names, i think we are okay.

"pickle" =>"read_pickle"
"json" => "read_json"

Reply:
yes i've corrected as you suggested